openmetadata-ui/src/main/resources/ui/src/components/ServiceConnectionDetails/ServiceConnectionDetails.component.tsx

```diff
@@ -169,25 +169,32 @@ const ServiceConnectionDetails = ({
           : {};
 
         return (
-          <div className="tw-w-1/2 tw-flex tw-nowrap tw-mb-3" key={key}>
-            <div className="tw-flex">
-              <p className="tw-text-gray-500 tw-m-0">{title || key}:</p>
-              <Tooltip position="bottom" title={description} trigger="hover">
-                <InfoCircleOutlined
-                  className="tw-mx-1"
-                  style={{ color: '#C4C4C4' }}
+          <Col span={12}>
+            <Row>
+              <Col span={8}>
+                <Space size={0}>
+                  <p className="text-grey-muted m-0">{key || title}:</p>
+                  <Tooltip
+                    position="bottom"
+                    title={description}
+                    trigger="hover">
+                    <InfoCircleOutlined
+                      className="tw-mx-1"
+                      style={{ color: '#C4C4C4' }}
```

Comment:
One thing for later, keep color codes in some constant file, rather than hardcoding at component level.